### ISSUE TITLE
error when upgrading to 2.9.889

### ISSUE THREAD

**Issue by mohas** [fluffy-mods/ModManager#85]
**Describe the bug**  
Upgraded the mod to version 2.9.889 and this happens
![error](https://user-images.githubusercontent.com/1816728/79533778-fd802800-808d-11ea-85b3-b4d1489c35c2.png)
tested 2.6.886 and 2.6.887, same issue
also mouse clicks have an offset (when you click somewhere, somewhere else is being click),
did not have this problem with 2.7.882

**Other mods**  
here are my modlist 
[ModsConfig.txt](https://github.com/fluffy-mods/ModManager/files/4491000/ModsConfig.txt)

here is the error log, the log is about 10mb so I zipped it most of it is repeated tho
[Player.zip](https://github.com/fluffy-mods/ModManager/files/4491037/Player.zip)

here is the gist of it:

Exception filling window for ModManager.Page_BetterModConfig: System.MissingMethodException: void Verse.ModContentPack..ctor(System.IO.DirectoryInfo,string,string,int,string)
  at ModManager.Manifest.SetVersion (System.Boolean fromAssemblies) [0x00023] in <3644494e6e614eeeb6af74efad297307>:0 
  at ModManager.Manifest.For (Verse.ModMetaData mod) [0x00331] in <3644494e6e614eeeb6af74efad297307>:0 
  at ModManager.ModButton_Installed.get_Manifest () [0x00006] in <3644494e6e614eeeb6af74efad297307>:0 
  at ModManager.ModButton_Installed.get_Requirements () [0x00000] in <3644494e6e614eeeb6af74efad297307>:0 
  at ModManager.ModButton.DoModIssuesIcon (UnityEngine.Rect canvas) [0x0000d] in <3644494e6e614eeeb6af74efad297307>:0 
  at ModManager.ModButton_Installed.DoModButton (UnityEngine.Rect canvas, System.Boolean alternate, System.Action clickAction, System.Action doubleClickAction, System.Boolean deemphasizeFiltered, System.String filter) [0x001db] in <3644494e6e614eeeb6af74efad297307>:0 
  at ModManager.Page_BetterModConfig.DoAvailableMods (UnityEngine.Rect canvas) [0x0014f] in <3644494e6e614eeeb6af74efad297307>:0 
  at ModManager.Page_BetterModConfig.DoWindowContents (UnityEngine.Rect canvas) [0x00111] in <3644494e6e614eeeb6af74efad297307>:0 
  at Verse.Window.InnerWindowOnGUI (System.Int32 x) [0x00165] in <fcd99b24cbb3485baa8263b0d6197784>:0 
 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

GUI Error: You are pushing more GUIClips than you are popping. Make sure they are balanced.
 
(Filename: C:\buildslave\unity\build\Modules/IMGUI/GUIClip.cpp Line: 580)

**Environment**  
 - OS: Windows 10
 
**Game**
 - 64 bits
 - Steam
 - 1.1.2552

**Comment by mohas** [comment]
I thought this is compatibility issue with my version of rim world but tested 2.6.886 and 2.6.887 to no avail

**Comment by FluffierThanThou** [comment]
make sure to use the most recent version of RimWorld. You appear to be using a very early version of 1.1 (1.1.2552 to be precise).

When in doubt, the version of the game my mods are built against are always listed in the release title on GitHub, as well as in the description. 

The version you're using is actually so old that I haven't build Mod Manager for it specifically (the earliest I built for was 1.1.2562), but some of the earlier 2.0 releases of Mod Manager will likely work.   I would think Mod Manager v2.6.886 (1.1.2575) should work, but you may have to go even further back. 

TLDR; **update RimWorld.** I cannot and will not offer support for outdated versions of RimWorld, sorry.
